test/e2e/nodelatencymonitor_test.go

```diff
@@ -0,0 +1,142 @@
+package e2e
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	"antrea.io/antrea/pkg/apis/stats/v1alpha1"
+	"antrea.io/antrea/pkg/features"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestNodeLatencyMonitor(t *testing.T) {
+	skipIfFeatureDisabled(t, features.NodeLatencyMonitor, true, false)
+	skipIfHasWindowsNodes(t) // as the feature does not have official Windows support
+
+	data, err := setupTest(t)
+	require.NoError(t, err, "Error when setting up test")
+
+	defer teardownTest(t, data)
+
+	nodes, err := data.clientset.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	require.NoError(t, err, "Failed to list nodes")
+
+	var isDualStack bool
+	for _, node := range nodes.Items {
+		if node.Spec.PodCIDR != "" && strings.Contains(node.Spec.PodCIDR, ":") {
+			isDualStack = true
+			break
+		}
+	}
```

Comment:
you should be using `clusterInfo.podV4NetworkCIDR` and `clusterInfo.podV4NetworkCIDR` to determine this

we already have a framework that performs the correct checks for you